app/controllers/idv/phone_errors_controller.rb

```diff
@@ -74,8 +74,8 @@ def track_event(type:)
     # rubocop:disable Naming/MemoizedInstanceVariableName
     def set_gpo_letter_available
       return @gpo_letter_available if defined?(@gpo_letter_available)
-      @gpo_letter_available ||= FeatureManagement.gpo_verification_enabled? &&
-                                !Idv::GpoMail.new(current_user).rate_limited?
+      policy = Idv::GpoVerifyByMailPolicy.new(current_user)
+      @gpo_letter_available ||= policy.send_letter_available?
```

Comment:
ditto about direct assignment
```suggestion
      @gpo_letter_available = policy.send_letter_available?
```